src/main/java/tv/zerator/ffs/api/v1/resources/EventResource.java

```diff
@@ -1,6 +1,7 @@
 package tv.zerator.ffs.api.v1.resources;
 
 import java.sql.SQLException;
+import java.util.*;
```

Comment:
Not recommended to use wildcard import in Java, most of the formatter / codestyle rule prefer unit import

Reply:
changed